addons/l10n_pl/data/account_tax_report_data.xml

```diff
@@ -221,292 +225,304 @@
             </record>
             <record id="account_tax_report_line_do_przeniesienia" model="account.report.line">
                 <field name="name">Podatek - Do przeniesienia</field>
-                <field name="aggregation_formula">PLTAXC.balance - PLTAXD.balance</field>
+                <field name="code">PLTAX</field>
+                <field name="expression_ids">
+                    <record id="account_tax_report_line_do_przeniesienia_formula" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">PLTAXC.balance - PLTAXD.balance - PLTAX_49.balance - PLTAX_50.balance</field>
+                    </record>
+                    <record id="tax_report_line_do_przeniesienia_carryover" model="account.report.expression">
+                        <field name="label">_carryover_balance</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">PLTAX.balance</field>
+                        <field name="subformula">if_above(EUR(0))</field>
+                        <field name="carryover_target">PODATEK__NADWYZKA_Z_POPRZEDNIEJ_DEKLARACJI._applied_carryover_balance</field>
+                    </record>
+                </field>
                 <field name="children_ids">
-                    <record id="account_tax_report_line_nad_naleznym" model="account.report.line">
-                        <field name="name">Podatek - Nadwyżka naliczonego nad należnym</field>
-                        <field name="aggregation_formula">PLTAXC.balance - PLTAXD.balance</field>
+                    <record id="account_tax_report_line_podatek_razem_c" model="account.report.line">
+                        <field name="name">Podatek - Razem C</field>
+                        <field name="code">PLTAXC</field>
+                        <field name="aggregation_formula">
+                            PLTAXC_04_16.balance +
+                            PLTAXC_05_18.balance +
+                            PLTAXC_06_20.balance +
+                            PLTAXC_09_24.formula +
+                            PLTAXC_10_26.balance +
+                            PLTAXC_11_28.balance +
+                            PLTAXC_12_32.balance +
+                            PLTAXC_12_33.balance +
+                            PLTAXC_01_34.balance
```

Comment:
You need to inline this. Otherwise, I think the display of this field is really weird on the form view

Reply:
Just checked it, it does not impact the display as far as I can tell. Still changed it to stay consistent with the rest